### PR TITLE
Add meta prop to Entity data table

### PR DESCRIPTION
**Pull request by maxiadlovskii**
## Motivation and Context
Usually, for Entity data table we use only prop list to get some data from the BE. Sometimes BE endpoints might include additional data that might be used to render cells. This PR extends PaginatedEntityTable and EntityDataTable to receive new prop `meta`. The meta data will be accessible from the column renderer functions and hook.

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [x] Refactoring (non-breaking change)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
/nocl

### CODE REVIEW

File: graylog2-web-interface/src/components/common/EntityDataTable/EntityDataTable.tsx [hunk 1]
```diff
@@ -143,7 +144,7 @@ const mergeColumnsRenderers = <Entity extends EntityBase>(columns: Array<Column>
   }));
 };
 
-type Props<Entity extends EntityBase> = {
+type Props<Entity extends EntityBase, Meta = unknown> = {
```
[the remark this answers is not in the document]
Comment (linuspahl) [reply graylog2-web-interface/src/components/common/EntityDataTable/EntityDataTable.tsx:147]: It looks like there are a few more case where we can pass on the `Meta` type, like  `columnRenderersByAttribute` and `mergeColumnsRenderers`

[the remark this answers is not in the document]
Comment (maxiadlovskii) [reply graylog2-web-interface/src/components/common/EntityDataTable/EntityDataTable.tsx:147]: With the default param, we making this property optional

[the remark this answers is not in the document]
Comment (linuspahl) [reply graylog2-web-interface/src/components/common/EntityDataTable/EntityDataTable.tsx:147]: That makes sense! I would just replace the usage of `any` in `useFetchEntities`.

[the remark this answers is not in the document]
Comment (maxiadlovskii) [reply graylog2-web-interface/src/components/common/EntityDataTable/EntityDataTable.tsx:147]: @linuspahl Replaced

**Review by linuspahl — APPROVED** [review]

**Pull request merged**